app/react/Metadata/components/specs/__snapshots__/ShowMetadata.spec.js.snap

```diff
@@ -79,6 +79,7 @@ exports[`Metadata should render without timeline by default 1`] = `
   <Connect(BaseFormatMetadata)
     entity={Object {}}
     groupGeolocations={false}
+    renderLabel={[Function]}
   />
 </div>
 `;
```

Comment:
can you please replace this full snapshot with a specific test for the timeline not being rendered? i know its not from the issue.

Reply:
Yeah, makes sense 👍